openapi.yaml

```diff
@@ -1221,7 +1221,7 @@ components:
           enum:
           - mix
           - fully-fashioned
-          - seamless
+          - integral
           - circular
           - straight
```

Comment:
Note that this enum is the very same as the one above, and may be factored and linked out, so we have a single place to update it.